presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -321,30 +319,27 @@ There are four selectors, that define which queries run in which resource group:
 
 * The first selector matches queries from ``bob`` and places them in the admin group.
 
-* The second selector matches queries from ``admin`` user group and places them in the admin group.
-
-* The third selector matches all data definition (DDL) queries from a source name that includes ``pipeline``
+* The second selector matches all data definition (DDL) queries from a source name that includes ``pipeline``
   and places them in the ``global.data_definition`` group. This could help reduce queue times for this
   class of queries, since they are expected to be fast.
 
-* The fourth selector matches queries from a source name that includes ``pipeline``, and places them in a
+* The third selector matches queries from a source name that includes ``pipeline``, and places them in a
   dynamically-created per-user pipeline group under the ``global.pipeline`` group.
 
-* The fifth selector matches queries that come from BI tools which have a source matching the regular
-  expression ``jdbc#(?<toolname>.*)`` and have client provided tags that are a superset of ``hipri``.
-  These are placed in a dynamically-created sub-group under the ``global.adhoc`` group.
-  The dynamic sub-groups are created based on the values of named variables ``toolname`` and ``user``.
-  The values are derived from the source regular expression and the query user respectively.
-  Consider a query with a source ``jdbc#powerfulbi``, user ``kayla``, and client tags ``hipri`` and ``fast``.
-  This query is routed to the ``global.adhoc.bi-powerfulbi.kayla`` resource group.
+* The fourth selector matches queries that come from BI tools which have a source matching the regular
+  expression ``jdbc#(?<toolname>.*)``, and have client provided tags that are a superset of ``hi-pri``.
+  These are placed in a dynamically-created sub-group under the ``global.pipeline.tools`` group. The dynamic
+  sub-group are created based on the named variable ``toolname``, which is extracted from the in the
```

Comment:
```suggestion
  sub-group is created based on the named variable ``toolname``, which is extracted from the 
```
I deleted "in the". Should this correctly be "which is extracted from the regular expression for source" or "which is extracted in the regular expression for source"? Change this to the correct meaning, please.

Reply:
The correct phrasing would be "which is extracted from the regular expression for source". This indicates that the variable toolname is derived or obtained from the source's regular expression.